lib/log_helper.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
Why don't place the test helper into the directory with tests?

Reply:
Not relevant anymore.